clusterloader2/cmd/clusterloader.go

```diff
@@ -297,31 +295,25 @@ func main() {
 		klog.Exitf("Error while preloading images: %v", err)
 	}
 
-	suiteSummary := &ginkgotypes.SuiteSummary{
-		SuiteDescription:           "ClusterLoaderV2",
-		NumberOfSpecsThatWillBeRun: len(testConfigPaths),
-	}
-	junitReporter := ginkgoreporters.NewJUnitReporter(path.Join(clusterLoaderConfig.ReportDir, "junit.xml"))
-	junitReporter.SpecSuiteWillBegin(ginkgoconfig.GinkgoConfig, suiteSummary)
-	testsStart := time.Now()
+	testReporter := test.CreateSimpleTestReporter(path.Join(clusterLoaderConfig.ReportDir, "junit.xml"), "ClusterLoaderV2")
+	testReporter.BeginTestSuite()
```

Comment:
Before submitt, can you please double check that this will also work fine if run for the suite?
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml#L2
is a good example

Reply:
Good catch - I didn't notice that we actually use `--testsuite` flag in our test jobs.

I will come back here with the results confirming it also works for this type of tests setup.

Reply:
I ran a manual test and it looks okay apart from the fact that some steps do not have a name in the [config](https://github.com/kubernetes/perf-tests/blob/2afae16bdd662ef636404cf165df864f90dc1b81/clusterloader2/testing/experimental/storage/pod-startup/config.yaml) that test suite leverages. Because of that, some of the reported test cases have the same identifier from my PR's point of view and Prow treats them as duplicates and filters them out.

I propose that we add names for every step in that config and implement a mechanism in CL2 that imposes providing a name for each of the steps. What do you think?

Reply:
For now, can we make a change in CL2 that if a step doesn't have name then a name is generated, e.g. something like: "Step N [autogenerated, please name your step]"? Can we do it in a separate CL? It should be fairly small change

We can start naming the steps later and maybe eventually start enforcing steps to have name.

Reply:
Sounds good to me; will come back with an appropriate PR here.

Reply:
PR with step names autogeneration: https://github.com/kubernetes/perf-tests/pull/1472.

Reply:
Done.